### PR TITLE
<Palette/> Neutral pallete layout should use two columns if text doesn't fit

### PR DESCRIPTION
**Pull request by carin91**
1) Replacing padding: "0px 0px 0px 0px" with padding: "0px" 
2) Adding new midSize boolean for widths under 1140px"
3) Adding new ternary selector for the number of columns
4) Adding new ternary selector for the number of rows (10 rows in the case of 2 columns based on figma mockups)

To fix: There is a duplicated RenderCategoyGrid component, used to render the palette outlet and used to render the tokenPicker in the tokenColorCard component, it is necessary a new file with this component.

### CODE REVIEW

File: src/components/cards/TokenColorCard/index.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ function RenderCategoryGrid(props: renderCategoryGridProps) {
       <Text
         type="title"
         size="medium"
-        padding="0px 0px 0px 0px"
+        padding="0px"
```
Comment (LinconOssa) [inline src/components/cards/TokenColorCard/index.tsx:47]: image
you can add this command to not put 0px

File: src/pages/people/outlets/palette/interface.tsx [hunk 2]
```diff
@@ -79,7 +79,7 @@ function RenderCategoryGrid(props: renderCategoryGridProps) {
       <Text
         type="title"
         size="medium"
-        padding="0px 0px 0px 0px"
+        padding="0px"
```
Comment (LinconOssa) [inline src/pages/people/outlets/palette/interface.tsx:82]: same here the padding

**Review by LinconOssa — CHANGES_REQUESTED** [review]

**Review by LinconOssa — APPROVED** [review]
meets the requirements

File: src/pages/people/outlets/palette/interface.tsx [hunk 3]
```diff
@@ -123,7 +117,8 @@ export function PaletteUI(props: PaletteUIProps) {
     handleReset,
   } = props;
 
-  const smallScreen = useMediaQuery("(max-width: 580px)");
+  const smallScreen = useMediaQuery("(max-width: 640px)");
```
[the remark this answers is not in the document]
Comment (carin91) [reply src/pages/people/outlets/palette/interface.tsx:120]: attended
image

[the remark this answers is not in the document]
Comment (cmarin001) [reply src/pages/people/outlets/palette/interface.tsx]: meet the requirements

**Pull request merged**